cirq-google/cirq_google/engine/engine_client.py

```diff
@@ -387,7 +387,8 @@ async def create_job_async(
         description: Optional[str] = None,
         labels: Optional[Dict[str, str]] = None,
         *,
-        run_name: str = "",
+        run_name: str | None = "",
+        snapshot_id: str | None = None,
```

Comment:
why the default value of `run_name` is `""` instead of None? Can we make `run_name` and `snapshot_id` identical -- both use None?